### PR TITLE
Termsandconditions

### PR DESCRIPTION
**Pull request by Kalyani790**

### CODE REVIEW

File: app/assets/translations/appConstants/bebbo/UZuz.ts [hunk 1]
```diff
@@ -448,6 +448,8 @@ const UZuz = {
               donateButton: "Bebboʼga moliyaviy koʻmak",
               contactUs: "Bizga yozish",
               aboutBebboDrawerMenu: "{{appName}} haqinda",
+              tNcTermsConditions: "Bebbo ilovasi foydalanish shartlarini",
```
Comment (Akhror) [inline app/assets/translations/appConstants/bebbo/UZuz.ts:451]: I would name it as tncCheckBoxText.

File: app/assets/translations/appConstants/bebbo/UZuz.ts [hunk 2]
```diff
@@ -448,6 +448,8 @@ const UZuz = {
...
+              tNcStaticContent: "Bebbo shaxsiy maʼlumotlarni toʻplamaydi va saqlamaydi. Siz kiritgan barcha maʼlumotlar telefoningizda saqlanadi.",
```
Comment (Akhror) [inline app/assets/translations/appConstants/bebbo/UZuz.ts:452]: I would name it as tncScreenContent

File: app/screens/Terms.tsx [hunk 3]
```diff
@@ -230,7 +216,7 @@ const Terms = ({ navigation }: Props): any => {
                       )}
                     </View>
                   </CheckboxItem>
-                  <LabelText>{t('tNccheckbox1')}</LabelText>
+                  <LabelText>{t('tNccheckbox2')} <CheckboxItemText onPress={goToTerms} style={styles.checkboxStyle}>{t('tNcTermsConditions')}</CheckboxItemText></LabelText>
```
Comment (Akhror) [inline app/screens/Terms.tsx:219]: Is this change intentional? tNccheckbox1 to tNccheckbox2

Comment (Kalyani790) [reply app/screens/Terms.tsx:219]: yes this is intentional as per new requirement.

File: app/assets/translations/appConstants/bebbo/UZuz.ts (hunk 2 above)
Comment (Kalyani790) [reply app/assets/translations/appConstants/bebbo/UZuz.ts:452]: Noted.

File: app/assets/translations/appConstants/bebbo/UZuz.ts (hunk 1 above)
Comment (Kalyani790) [reply app/assets/translations/appConstants/bebbo/UZuz.ts:451]: Noted.
